src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,34 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+//CSS
+import './button.css';
+
+function Button(props) {
+  const { label, handleClick, className, id } = props;
+  return (
+    <button className={className} onClick={() => handleClick(id, 1)}>
```

Comment:
inline function onClick, why are you passing 1 as a 2nd parameter it doesn't look right